CRM/Admin/Form/Job.php

```diff
@@ -156,6 +159,13 @@ public function setDefaultValues() {
 
     CRM_Core_DAO::storeValues($dao, $defaults);
 
+    // CRM-17686
+    if (!empty($dao->scheduled_run_date)) {
+      $ts = strtotime($dao->scheduled_run_date);
```

Comment:
timestamp is UTC adjusted and stored internally as a timestamp, but is returned by a query in the same format as datetime.